zebra/zebra_rib.c

```diff
@@ -1156,16 +1184,24 @@ void rib_uninstall_kernel(struct route_node *rn, struct route_entry *re)
 
 	/*
 	 * Make sure we update the FPM any time we send new information to
-	 * the kernel.
+	 * the dataplane.
 	 */
 	hook_call(rib_update, rn, "uninstalling from kernel");
-	switch (kernel_route_rib(rn, p, src_p, re, NULL)) {
+	switch (dplane_route_delete(rn, re)) {
```

Comment:
Why do we need a dplane_route_delete?  Why is kernel_route_rib insufficient for this?